connector_prestashop/models/sale_order/importer.py

```diff
@@ -117,6 +118,31 @@ def _rule_global(self, record, mode):
                                  'because it has not been paid since %d '
                                  'days' % (order_id, max_days))
 
+    def _rule_state(self, record, mode):
```

Comment:
just to respect the same signature. Not mandatory indeed, but if someone wants to customize it further based also on payment mode is feasible with it.